connectors/src/connectors/confluence/lib/internal_ids.ts

```diff
@@ -7,7 +7,7 @@ Internal IDs are Confluence IDs with a prefix `confluence-space-` or `confluence
 
 enum ConfluenceInternalIdPrefix {
   Space = "confluence-space-",
-  Page = "confluence-page",
+  Page = "confluence-page-",
```

Comment:
gonna add this to the cleaning migration 😬

Reply:
actually I had scaled down the worker so `select * from data_sources_documents where document_id like 'confluence-page[0-9]%' limit 1;` returns nothing 🥵

Reply:
on a encore eu de la chance